sources/packages/web/src/views/student/financial-aid-application/Applications.vue

```diff
@@ -31,6 +31,21 @@
     <v-icon size="25" class="mr-2">mdi-text-box-plus</v-icon>
     Start New Application
   </v-btn>
+  <v-btn
```

Comment:
Since @ann-aot changes to application summary will be done inside the sprint and we can demo this accessing directly the URL, I would consider not adding this.
If you think that is still worth you can keep it, but we need to remove it as soon it is integrated into the application summary list, up to you 😉 .